packages/client/src/__tests__/integration/happy/output-directory/test.ts

```diff
@@ -0,0 +1,19 @@
+import { generateInFolder } from '../../../../utils/generateInFolder'
+
+beforeAll(async () => {
+  await generateInFolder({
+    projectDir: __dirname,
+    useLocalRuntime: true,
+    transpile: false,
+    useBuiltRuntime: false,
+    relativeOutputDir: './prismaClient',
+  })
+})
+
+test('connect to client generated 1 directory level', async () => {
+  const { PrismaClient } = await import('./prismaClient')
```

Comment:
```suggestion
  // @ts-ignore
  const { PrismaClient } = await import('./prismaClient')
```

Reply:
@arthurfiorette nope, that won't work. It will fix the CI but break the local workflow as soon as you run that test at least once.

Reply:
ok